supabase/migrations/20260725163259_slim_audit_logs_skip_bookkeeping.sql

```
@@ -0,0 +1,208 @@
+-- Slim audit_logs: do not store internal app_versions upload/migrate bookkeeping.
+-- Capgo-EU evidence: r2_path/storage_provider/manifest pipeline updates were hundreds
+-- of MB of TOAST with no user-facing audit value. Soft-delete and real edits stay.
+-- Historical cleanup is intentionally NOT in this migration (avoids one-shot WAL/lock
```

SQLFluff (4.2.2)

[error] Line is too long (86 > 80).

(LT05)